test/with_dummyserver/test_socketlevel.py

```diff
@@ -1351,7 +1351,7 @@ def socket_handler(listener):
         with mock.patch("urllib3.util.ssl_.SSLContext", lambda *_, **__: context):
             self._start_server(socket_handler)
             with HTTPSConnectionPool(self.host, self.port) as pool:
-                with pytest.raises(MaxRetryError):
+                with pytest.raises(Exception):
```

Comment:
Not a huge fan of asserting `Exception`, means this error could change any time and we wouldn't notice

Reply:
I usually avoid Pokemon exception handling too, but what we really want to test is that `load_default_certs` was called, and this continues to be true with this pull request.

Regarding the exception, we know that without a correct `SSLContext` things are going to fail at some point, and we currently fail with a `TypeError` instead of a `MaxRetryError`. Since we don't care about the actual exception type, I thought using `Exception` was fine.

Would you prefer using `with contextlib.suppress(Exception)`?

Reply:
No if all we care about is an exception is raised this is fine.